src/MUIDataTable.js

```diff
@@ -1360,15 +1374,16 @@ class MUIDataTable extends React.Component {
         elevation={this.options.elevation}
         ref={this.tableContent}
         className={classnames(classes.paper, className)}>
-        {selectedRows.data.length && this.options.disableToolbarSelect !== true ? (
+        {selectedRows.data.length > 0 && this.options.selectToolbarPlacement !== 'none' && (
           <TableToolbarSelect
             options={this.options}
             selectedRows={selectedRows}
             onRowsDelete={this.selectRowDelete}
             displayData={displayData}
             selectRowUpdate={this.selectRowUpdate}
           />
-        ) : (
+        )}
+        {(selectedRows.data.length === 0 || ['above-toolbar', 'none'].includes(this.options.selectToolbarPlacement)) &&
```

Comment:
It would be better to create a constant with this array of options, and place it toward the top of the file. That way, it's easier to alter the names in a single location without forgetting anything, and it will document better what these options are.